src/lib/y2partitioner/actions/controllers/filesystem.rb

```diff
@@ -369,19 +433,50 @@ def current_value_for(attribute)
           when :mount_by
             filesystem.mount_by
           when :mount_point
-            filesystem.mount_point
+            filesystem.mount_path
           when :label
             # Copying the label from the filesystem in the disk looks unexpected
             new?(filesystem) ? filesystem.label : nil
           end
         end
 
-        def before_set_mount_point
+        # Values to apply for the selected role
+        #
+        # @return [Hash]
+        def role_values
+          fs_type = mount_path = mount_by = nil
+
+          case role
+          when :swap
+            part_id = Y2Storage::PartitionId::SWAP
+            fs_type = Y2Storage::Filesystems::Type::SWAP
+            mount_path = "swap"
+            mount_by = Y2Storage::Filesystems::MountByType::DEVICE
```

Comment:
Not sure: Does this always enforce MountByType::DEVICE for swap? Is there still a chance to set LABEL or UUID despite this? Is this only a default, or is it the only possible value?

Reply:
It is only a default. It could be changed using the button for "fstab options".